src/Components/__stories__/AuthForm.story.tsx

```diff
@@ -0,0 +1,29 @@
+import { storiesOf } from "@storybook/react"
+import React from "react"
+// import { graphql } from "react-relay"
+
+// import { RootQueryRenderer } from "../../Relay/RootQueryRenderer"
+import AuthForm from "../Authorization/AuthForm"
+
+// maybe can use this to actually test auth?
+// function ArtworkExample(props: { artworkID: string }) {
+//   return (
+//     <RootQueryRenderer
+//       query={graphql`
+//         query ArtworkQuery($artworkID: String!) {
+//           artwork(id: $artworkID) {
+//             ...Artwork_artwork
+//           }
+//         }
+//       `}
+//       variables={{ artworkID: props.artworkID }}
+//       render={readyState =>
+//         readyState.props && <Artwork {...readyState.props as any} />}
+//     />
+//   )
+// }
```

Comment:
I wouldn't add the actual submission logic to reaction. There are so many things that can't be taken care of on reaction - how to launch a modal from old-school plain JS or coffee script, possible style conflict, where to redirect after submission, etc. I think reaction should stay as a style guide like the Bootstrap that allows us to quickly build up a UI with low-level components _on Force_.

WDYT? @l2succes @xtina-starr @eessex

Reply:
I would agree but by using Relay containers and colocating data requirements with components, we've already made reaction more than just a UI library, it handles the data/model layer as well. 

So IMO, encapsulating submission logic in reaction is fine. However, I do get the argument to handle data submission and redirecting in Force. Basically I think either is fine, as long as we don't have multiple implementations of site-wide login and sign up forms lying around.